webrender/src/picture.rs

```diff
@@ -520,8 +532,14 @@ impl TileCache {
                 let key = (px.round() as i32, py.round() as i32);
 
                 let mut tile = match old_tiles.remove(&key) {
-                    Some(tile) => tile,
-                    None => Tile::new(),
+                    Some(tile) => {
+                        tile
+                    }
+                    None => {
+                        let id = TileId(self.next_id);
+                        self.next_id += 1;
+                        Tile::new(id)
```

Comment:
I think it would be a bit cleaner to make next_id a method and then this could just be Tile::new(self.next_id()).

Reply:
Done